src/main/java/com/toptrumps/db/RoundWinnersDAO.java

```diff
@@ -0,0 +1,5 @@
+package com.toptrumps.db;
+
+public interface RoundWinnersDAO {
+    boolean create();
```

Comment:
public boolean... ??

Reply:
method returns a boolean when implemented rather than being void just as standard practice to show that an update has been successful